.gitignore

```diff
@@ -0,0 +1 @@
+*__pycache__*
```

Comment:
I'm just trying to ignore python cache files from the commits. I think this will do it i.e. https://stackoverflow.com/questions/1470572/ignoring-any-bin-directory-on-a-git-project